rpm/containerd.spec

```diff
@@ -121,11 +121,13 @@ install -D -m 0755 bin/* %{buildroot}%{_bindir}
 install -D -m 0644 %{S:1} %{buildroot}%{_unitdir}/containerd.service
 install -D -m 0644 %{S:2} %{buildroot}%{_sysconfdir}/containerd/config.toml
 
-# install manpages
-install -d %{buildroot}%{_mandir}/man1
-install -p -m 644 man/*.1 $RPM_BUILD_ROOT/%{_mandir}/man1
-install -d %{buildroot}%{_mandir}/man5
-install -p -m 644 man/*.5 $RPM_BUILD_ROOT/%{_mandir}/man5
+# install manpages, taking into account that not all sections may be present
+for i in $(seq 1 8); do
```

Comment:
I see that we used to fill man 1 and 5 and that we're now doing 1 and 8, was this change intended?

Reply:
Yes; the man-pages were in the wrong section (1 instead of 8), this was fixed upstream, but now the packaging script here didn't take section 8 into account.

I changed it to take all sections (1..8) into account, so that we won't have to update this again if new man pages are added in future that are in different sections